### PR TITLE
Nbis

### PR DESCRIPTION
**Pull request by Juke34**
Fix path to the correct lecture slides.
+ some typo correction

### CODE REVIEW

File: docs/nbis_annotation/schedule.md [hunk 1]
```diff
@@ -14,22 +14,23 @@ Genome Annotation Service
...
-**09.00-09.15:** Summary of yesterday’s exercise  
-**09.15-09.30:**  Lecture: Introduction to manual curation  
-**10.00-12.00:** [Practical 3: Manual curation(incl. coffee break)](practical_session/practical3_manualCuration.md)  
-**11.30-12.30:** Lunch   
-**12.30-13.00:** [Lecture: Functional annotation](https://github.com/SGBC/course/blob/master/docs/nbis_annotation/slides/Structural_annotation.pptx)    
-**13.00-15.30:** [Practical 4: Functional annotation (incl. coffee break)](practical_session/practical4_funcAnnotInterp.md)  
-**15.30-16.00:** Wrap-up   
+**09.00-9.15** Summary of yesterday’s exercise  
+**09.15-09.30** Lecture: Introduction to manual curation  
+**10.00-12.00** [Practical 3: Manual curation(incl. coffee break)](practical_session/ExerciseManCuration)
```
Comment (HadrienG) [inline docs/nbis_annotation/schedule.md]: look at your file names. this change breaks the link

File: docs/nbis_annotation/schedule.md [hunk 2]
```diff
@@ -14,22 +14,23 @@ Genome Annotation Service
...
-**11.30-12.30:** Lunch   
-**12.30-13.00:** [Lecture: Functional annotation](https://github.com/SGBC/course/blob/master/docs/nbis_annotation/slides/Structural_annotation.pptx)    
-**13.00-15.30:** [Practical 4: Functional annotation (incl. coffee break)](practical_session/practical4_funcAnnotInterp.md)  
-**15.30-16.00:** Wrap-up   
+**09.00-9.15** Summary of yesterday’s exercise  
+**09.15-09.30** Lecture: Introduction to manual curation  
+**10.00-12.00** [Practical 3: Manual curation(incl. coffee break)](practical_session/ExerciseManCuration)  
+**11.30-12.30** Lunch   
+**12.30-13.00** [Lecture: Functional annotation](slides/Functional_annotation.pptx)  
+**13.00-15.30** [Practical 4: Functional annotation (incl. coffee break)](practical_session/ExerciseFuncAnnotInterp)
```
Comment (HadrienG) [inline docs/nbis_annotation/schedule.md]: breaks the link too

**Review by HadrienG — CHANGES_REQUESTED** [review]

File: docs/nbis_annotation/schedule.md (hunk 2 above)
Comment (HadrienG) [reply docs/nbis_annotation/schedule.md]: your tutorials are **not** named `ExerciseManCuration` and `ExerciseFuncAnnotInterp` but rather practical3_manualCuration.md` and `practical4_funcAnnotInterp.md`, respectively. Rename your files or change the schedule

**Comment by Juke34** [comment]
It should be fine now !

**Review by HadrienG — APPROVED** [review]

**Pull request merged**